### PR TITLE
[improve]  add JdbcSourceSchema extends SourceSchema

### PR DESCRIPTION
**Pull request by bingquanzhao**
add jdbc source schema extends source schema .Making SourceSchema more than just a JDBC-related implementation

### CODE REVIEW

File: flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/SourceSchema.java [hunk 1]
```diff
@@ -33,57 +31,23 @@
...
-        primaryKeys = new ArrayList<>();
-        try (ResultSet rs = metaData.getPrimaryKeys(databaseName, schemaName, tableName)) {
-            while (rs.next()) {
-                String fieldName = rs.getString("COLUMN_NAME");
-                primaryKeys.add(fieldName);
-            }
-        }
     }
 
-    public abstract String convertToDorisType(String fieldType, Integer precision, Integer scale);
```
Comment (JNSimba) [inline flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/SourceSchema.java:85]: Do we need to delete this too?

File: flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/JdbcSourceSchema.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
+package org.apache.doris.flink.tools.cdc;
```
Comment (JNSimba) [inline flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/JdbcSourceSchema.java:1]: add license header

**Comment by JNSimba** [comment]
please rebase master, there is a conflict

**Comment by bingquanzhao** [comment]
> please rebase master, there is a conflict

👌

**Pull request closed without merging**